services/tables/src/main/java/com/linkedin/openhouse/tables/repository/impl/InternalRepositoryUtils.java

```diff
@@ -181,6 +181,11 @@ private static long safeParseLong(String keyName, Map<String, String> megaProps)
    * in HTS and client-visible table location.
    */
   static String getSchemeLessPath(String rawPath) {
-    return URI.create(rawPath).getPath();
+    URI uri = URI.create(rawPath);
```

Comment:
Also added this to allow for locations with and without the prefix to be equal. However, also feels a bit hacky -- is this the best way to go about it or does anyone see a better solution?